react/src/js/react/App.jsx

```diff
@@ -1,23 +1,24 @@
-/**
- * Entry-point for the react code in the app.
- * Allows us to map various URLs to page-components
- */
-
 import React from "react";
 import Reflux from "reflux";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import PageNotFound from "./help/PageNotFound";
+import PostPage from "./posts/PostPage";
 import Header from "./misc/Header";
 
+
+/**
+ * Entry-point for the react code in the app.
+ * Allows us to map various URLs to page-components
+ */
 export default class App extends Reflux.Component {
     render() {
         return (
             <div>
                 <Header />
                 <BrowserRouter>
                     <Switch>
-                        <Route path="*" component={PageNotFound} />
```

Comment:
I wouldn't remove that, since if you go to an invalid URL you would want to see a 404

Reply:
okay, then follow up question, why are we using switch, it can only render one react component? so I have to choose one?

Reply:
just kidding I am dumb, I understand switch now